benches/tinystr.rs

```diff
@@ -78,6 +78,40 @@ fn construct_from_str(c: &mut Criterion) {
     bench_block!(c, "construct_from_str", cfs);
 }
 
+fn construct_from_u8(c: &mut Criterion) {
+    macro_rules! cfu {
+        ($r:ty) => {
+            |b, inputs: &Vec<&str>| {
```

Comment:
Is there any reason this uses `&Vec`? `&Vec` is always useless (so should just be `&[&str]`).